custom_components/weenect/device_tracker.py

```diff
@@ -129,7 +129,7 @@ def __init__(
         tracker_id: int,
     ):
         super().__init__(coordinator, tracker_id)
-        self._attr_icon = "mdi:paw"
+        self._attr_icon = "mdi:tag"
```

Comment:
This would be a breaking change. To avoid this we could change the icon based on the response from the backend. There is `"type": "pet2",` in there. If the type starts with `pet` we can stay with the paw

Reply:
implemented